lib/component.js

```diff
@@ -163,3 +158,5 @@ title: ${componentTitle}
     }
   });
 }
+
+init();
```

Comment:
Remember to `.catch` any async errors thrown by this function. Node will track rejected promises, but it's fairly inconsistent - right now it's a warning, but eventually it'll be a hard error.